projects/uitdatabank/docs/search-api/sorting.md

```diff
@@ -71,17 +72,23 @@ GET /events/?sort[modified]=desc
 
 The events that were modified recently will appear first.
 
-### score
+### completeness
```

Comment:
To be consistent with the now alphabetical order, this block should be moved between the sorting parameter `availableTo` and `created`